test_runner/src/main/kotlin/ftl/util/StopWatch.kt

```diff
@@ -26,7 +26,10 @@ suspend fun measureTime(block: suspend () -> Unit) = StopWatch().run {
     check()
 }
 
-data class Duration(val seconds: Long)
+data class Duration(
+    val seconds: Long,
+    val nanos: Int? = null
```

Comment:
If seconds is in Long, should not nanos be Long too? It potentially can be a larger number.

Reply:
this is copy-pasted from google api